code/game/turfs/turf.dm

```diff
@@ -417,8 +417,15 @@
 /turf/proc/acid_melt()
 	return
 
-/turf/handle_fall()
+/turf/handle_fall(atom/falling)
 	if(has_gravity(src))
+		if(istype(falling, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = falling
+			// isn't it awful that this is where this is scoped
+			// ah well
+			if(ismachineperson(H) && prob(1) && prob(50))  // 0.5 % chance
```

Comment:
```suggestion
			if(ismachineperson(H) && prob(1) && prob(50))  // 0.5 % chance
```
Doesn't prob(0.5) work? I forget

Reply:
I guess? I thought I remembered there being issues with it, but it seems to compile